README.markdown

```diff
@@ -390,6 +390,10 @@ An optional Lua table can be specified as the last argument to this method to sp
     Specifies whether to perform SSL certificate verification during the
 SSL handshake if the `wss://` scheme is used.
 
+* `headers`
+
+    Specifies custom headers to be sent in the handshake request. The table is expected to contain strings in the format `{"a-header: a header value", "another-header: another header value"}`.
```

Comment:
Needs to add a note for when this feature was first added.

Reply:
Don't see a sample of that anywhere in the README - please clarify.

Reply:
@salazar One example is right under this paragraph:

> The SSL connection mode (`wss://`) requires at least `ngx_lua` 0.9.11 or OpenResty 1.7.4.1.

Reply:
@agentzh Could you please instruct me on what exactly to put?

Reply:
@salazar Never mind, I'll add it myself then. It would be much easier than the back and forth here :)